install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -152,6 +152,9 @@ spec:
         {{- end }}
         {{- if .Values.global.trustDomain }}
           - --trust-domain={{ .Values.global.trustDomain }}
+        {{- end }}
+        {{- if .Values.global.meshUID }}
```

Comment:
I asked in the other docs - what does "U" mean ? meshID I understand. UID - not so much.

Reply:
(normally it means user - but that doesn't make sense, and let's not redefined the UID from unix)

Reply:
UID = unique ID. That's also in common usage.
I will comment further in the doc.